lib/actions/help.js

```diff
@@ -70,11 +70,11 @@ help.usage = function () {
   let args = '';
 
   if (this._arguments.length > 0) {
-    args = this._arguments.map(formatArg).join(' ');
+    args = this._arguments.map(formatArg).join(' ') + ' ';
```

Comment:
Why move the trailing space here?

Reply:
Just for pretty output :smiley: Without trailing space the output will be like:
```
Usage:
  yo order:app [<a>] [<b>][options]
```

If there are no supported arguments in the generator and you do not change line 77 (``let out = `yo ${name} ${options} ${args}`;``), then output will be like (with two spaces between):

```
Usage:
  yo order:app  [options]
```